easybuild/easyblocks/p/perl.py

```diff
@@ -45,6 +45,8 @@ class EB_Perl(ConfigureMake):
     def extra_options():
         """Add extra config options specific to Perl."""
         extra_vars = {
+            'force_scripts_installdir': [False, "Force all scripts to be installed in the installation bin directory "
```

Comment:
`False` is backwards compatible, `True` would automatically solve this problem and probably does no harm...but I can't be sure.

Reply:
@ocaisa I see no reason to not always do this... scripts to into `<prefix>/bin` by default anyway, right?

Reply:
I'd consider that any of these scripts not ending up in `$EBROOTPERL/bin` is a bug and therefore I wouldn't worry about backwards compatibility. I wouldn't make this change optional either. @boegel ?

Reply:
@ocaisa https://github.com/ocaisa/easybuild-easyblocks/pull/16